src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -674,7 +674,7 @@ namespace cryptonote
     if (prune_blockchain)
     {
       // display a message if the blockchain is not pruned yet
-      if (m_blockchain_storage.get_current_blockchain_height() > 1 && !m_blockchain_storage.get_blockchain_pruning_seed())
+      if (!m_blockchain_storage.get_blockchain_pruning_seed())
       {
         MGINFO("Pruning blockchain...");
         CHECK_AND_ASSERT_MES(m_blockchain_storage.prune_blockchain(), false, "Failed to prune blockchain");
```

Comment:
I take it this prune_blockchain() will be a no-op if height <= 1 ?

Reply:
Yes.